pcsx2/GS/Renderers/HW/GSRendererHW.cpp

```diff
@@ -1927,7 +1950,9 @@ void GSRendererHW::Draw()
 			DetectDoubleHalfClear(no_rt, no_ds);
 	}
 
-	m_process_texture = PRIM->TME && !(PRIM->ABE && m_context->ALPHA.IsBlack() && !m_cached_ctx.TEX0.TCC);
+	m_process_texture = PRIM->TME && !(PRIM->ABE && m_context->ALPHA.IsBlack() && !m_cached_ctx.TEX0.TCC) &&
+		((m_cached_ctx.TEX0.TCC && (m_cached_ctx.TEST.ATE || (m_cached_ctx.FRAME.FBMSK & 0xFF000000u) != 0xFF000000u)) || !IsCd());
```

Comment:
would it not be better to move up (and maybe modify) the req_color and req_alpha checks from below where it's doing the src lookup?